glibc MIPS implements __WIFSTOPPED and __WIFSIGNALED as static inline
functions (SIGRTMAX 127 handling), not usable in constant expressions.

Signed-off-by: René Rebe <rene@exactco.de>

--- boost_1_92_0/boost/process/v1/detail/posix/is_running.hpp.vanilla	2026-10-13 18:42:52.795053502 +0200
+++ boost_1_92_0/boost/process/v1/detail/posix/is_running.hpp	2026-10-13 18:54:13.257625975 +0200
@@ -17,9 +17,14 @@ namespace boost { namespace process { BO
 // This bit arrangement of status codes is not guaranteed by POSIX, but (according to comments in
 // the glibc <bits/waitstatus.h> header) is the same across systems in practice.
 constexpr int still_active = 0x017f;
+// glibc MIPS implements __WIFSTOPPED and __WIFSIGNALED as non-constexpr inline functions
+#if !defined(__mips__)
 static_assert(WIFSTOPPED(still_active), "Expected still_active to indicate WIFSTOPPED");
+#endif
 static_assert(!WIFEXITED(still_active), "Expected still_active to not indicate WIFEXITED");
+#if !defined(__mips__)
 static_assert(!WIFSIGNALED(still_active), "Expected still_active to not indicate WIFSIGNALED");
+#endif
 static_assert(!WIFCONTINUED(still_active), "Expected still_active to not indicate WIFCONTINUED");
 
 inline bool is_running(int code)
--- boost_1_92_0/boost/process/v2/exit_code.hpp.vanilla	2026-10-13 18:42:52.803231439 +0200
+++ boost_1_92_0/boost/process/v2/exit_code.hpp	2026-10-13 18:54:13.273760796 +0200
@@ -73,9 +73,14 @@ typedef int native_exit_code_type;
 namespace detail
 {
 constexpr native_exit_code_type still_active = 0x17f;
+// glibc MIPS implements __WIFSTOPPED and __WIFSIGNALED as non-constexpr inline functions
+#if !defined(__mips__)
 static_assert(WIFSTOPPED(still_active), "Expected still_active to indicate WIFSTOPPED");
+#endif
 static_assert(!WIFEXITED(still_active), "Expected still_active to not indicate WIFEXITED");
+#if !defined(__mips__)
 static_assert(!WIFSIGNALED(still_active), "Expected still_active to not indicate WIFSIGNALED");
+#endif
 static_assert(!WIFCONTINUED(still_active), "Expected still_active to not indicate WIFCONTINUED");
 }
 
